src/pages/editor/EditorInstitution.tsx

```diff
@@ -101,3 +153,18 @@ export const EditorInstitution = () => {
     </>
   );
 };
+
+interface InstitutionUserItemProps {
+  user: InstitutionUser;
+}
+
+const InstitutionUserLink = ({ user }: InstitutionUserItemProps) => {
+  const fullName = `${user.givenName} ${user.familyName}`;
```

Comment:
`user-helpers.tsx` har en hjelpefunksjon. Den gjør ikke så mye, men greit å gjenbruke allikevel :) 

```suggestion
  const fullName = getFullName(user.givenName, user.familyName);
```